src/ert/simulator/batch_simulator_context.py

```diff
@@ -232,6 +261,13 @@ def job_status(self, iens: int) -> Optional["JobStatus"]:
         }
         return state_to_legacy[self._scheduler._jobs[iens].state]
 
+    def is_job_completed(self, iens: int) -> bool:
```

Comment:
Do we have an Everest PR that proves that it is possible to get rid of JobStatus by using these functions?

Reply:
https://github.com/equinor/everest/pull/62